wagtail_transfer/field_adapters.py

```diff
@@ -372,14 +372,15 @@ class AdapterRegistry:
     def __init__(self):
         self._scanned_for_adapters = False
         self.adapters_by_field_class = {}
-    
+
     def _scan_for_adapters(self):
         adapters = dict(self.BASE_ADAPTERS_BY_FIELD_CLASS)
 
         for fn in hooks.get_hooks('register_field_adapters'):
             adapters.update(fn())
-    
+
         self.adapters_by_field_class = adapters
+        self._scanned_for_adapters = True
```

Comment:
This is a small change that was previously missing. I mentioned to @jacobtoppm that I'd drop it in while passing by